src/components/core/NavigationTab/NavigationTabs.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactNode, useMemo } from 'react';
+import { useRouteMatch } from 'react-router-dom';
+
+const createGetter = function <T>(r: T, url: string) {
+  return (key: keyof T) => `${url}${r[key]}`;
+};
+
+type Result<T> = Record<keyof T, { to: string; value: string }>;
+
+export interface NavigationTabsProps<T> {
```

Comment:
Generic does not infer it needs to be an object.

Reply:
I don't understand

Reply:
The issue was resolved in a call